code/modules/client/preference/loadout/loadout_uniform.dm

```diff
@@ -61,17 +61,17 @@
 /datum/gear/uniform/skirt/job/viro
 	display_name = "skirt, virologist"
 	path = /obj/item/clothing/under/rank/virologist/skirt
-	allowed_roles = list("Chief Medical Officer","Medical Doctor")
+	allowed_roles = list("Chief Medical Officer","Medical Doctor" , "Virologist")
```

Comment:
Yeah, doctor shouldn't really have a skirt that's explicitly for the virologist. Neither should the CMO, really.